src/megameklab/com/printing/PrintMech.java

```diff
@@ -366,7 +386,7 @@ protected void drawArmorStructurePips() {
                 element = getSVGDocument().getElementById("isPips" + mech.getLocationAbbr(loc));
                 if (null != element) {
                     addPips(element, mech.getOInternal(loc),
-                            (loc == Mech.LOC_HEAD) || (loc == Mech.LOC_CT) || (loc == Mech.LOC_CLEG));
```

Comment:
The diff shown here leaves off the relevant condition that skips this block when loc > Mech.LOC_HEAD.